packages/vega-typings/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "vega-typings",
+  "version": "0.3.53",
+  "description": "Typings for Vega.",
+  "types": "types",
+  "repository": "vega/vega",
+  "author": "Dominik Moritz <domoritz@gmail.com>",
+  "keywords": [
+    "vega",
+    "typescript",
+    "typings"
+  ],
+  "license": "BSD-3-Clause",
+  "devDependencies": {
+    "dtslint": "^0.4.2",
```

Comment:
Although only used by vega-typings at the moment, I'm fine having these devDependencies moved to the top-level package.json file for consistency across packages and easier updates to the development environment.